### PR TITLE
Make Non-Toxic Water Stop Glowing

### PR DESCRIPTION
**Pull request by Venuska1117**
# About the pull request

This PR make non-toxic water stop glowing, it kinda don't make sense, that on some maps, where water cover essential points is just GLOWING, now water will not glow unless its intended to be glowing.

# Explain why it's good for the game
Water should not glow in darkness (unless its intended like on trijent dam)

# Testing Photographs and Procedure
waterchange-ezgif com-video-to-gif-converter

# Changelog
:cl:
fix: Make non-toxic water stop glowing.
/:cl:

### CODE REVIEW

File: code/game/turfs/floors/desert.dm [hunk 1]
```diff
@@ -114,7 +114,7 @@
 			set_light(2)
 			icon = 'icons/turf/floors/desert_water_toxic.dmi'
 		if(0)
-			set_light(1)
+			set_light(0)
```
Comment (harryob) [inline code/game/turfs/floors/desert.dm:117]:
```suggestion
```

File: code/game/turfs/floors/desert.dm [hunk 2]
```diff
@@ -224,7 +224,7 @@
...
```
Comment (harryob) [inline code/game/turfs/floors/desert.dm:227]:
```suggestion
```

File: code/game/turfs/open.dm [hunk 3]
```diff
@@ -168,6 +168,7 @@
 /turf/open/river
 	can_bloody = FALSE
 	supports_surgery = FALSE
+	var/toxic = 0
```
Comment (harryob) [inline code/game/turfs/open.dm:171]: this won't be doing anything, as the toxic water behavior only exists in update_icon of /turf/open/desert/desert_shore, /turf/open/desert/cave/cave_shore and /turf/open/gm/river/desert. you need to remove the set_light(1) in all these cases

**Review by harryob — CHANGES_REQUESTED** [review]

File: code/game/turfs/open.dm (hunk 3 above)
Comment (Venuska1117) [reply code/game/turfs/open.dm:171]: i tryied by just placing "set_light = 0" but it give error back instead

File: code/game/turfs/floors/desert.dm (hunk 1 above)
Comment (Venuska1117) [reply code/game/turfs/floors/desert.dm:117]: i cannot remove them, because it will keep light ON

File: code/game/turfs/open.dm [hunk 4]
```diff
@@ -12,6 +12,7 @@
 	var/scorchable = FALSE //if TRUE set to be an icon_state which is the full sprite version of whatever gets scorched --> for border turfs like grass edges and shorelines
 	var/scorchedness = 0 //how scorched is this turf 0 to 3
 	var/icon_state_before_scorching //this is really dumb, blame the mappers...
+	var/set_light //this remove or enable light (0 to disable)
```
Comment (harryob) [inline code/game/turfs/open.dm:15]: this doesn't do anything still, you can remove it and all your changes to open.dm and your code will work just as well

**Review by harryob — CHANGES_REQUESTED** [review]

File: code/game/turfs/open.dm (hunk 4 above)
Comment (Venuska1117) [reply code/game/turfs/open.dm]: it really was this simple?...

**Review by harryob — APPROVED** [review]

**Pull request merged**